language/types_values_variables.md

```diff
@@ -142,16 +146,17 @@ may appear more than once in the hierarchy (e.g. a `Scalar` is both `Any` and `D
        |  |     |- (Integer with range inside another Integer)
        |  |  |- Float[from, to]
        |  |     |- (Float with range inside another Float)
+       |  |  |- Timespan
+       |  |  |- Timestamp
        |  |
        |  |- String[from, to]
        |  |  |- Enum[*strings]
        |  |  |- Pattern[*patterns]
        |  |
        |  |- Boolean
        |  |- Regexp[pattern_string]
-       |
-       |- SemVer
-       |- SemVerRange
+       |  |- SemVer
+       |  |- SemVerRange
```

Comment:
Oh, I didn't realize that `Data` was constrained to json data types. That would rule out `Pattern` and `Regexp` as well. If that's what it is, then why not remove `Scalar` from the tree beneath `Data` and instead add `Integer`, `Float`, `String`, and `Boolean`? A `Scalar` will then not be constrained to json.